renpy.el

```diff
@@ -77,10 +79,12 @@
 	    ;; Images.
 	    (image-keyword
 	     (seq line-start "image" symbol-end))
-	    ;; Define or default.  Allow indentation for use in explicit init
+	    ;; Define and default.  Both allow indentation for use in explicit init
 	    ;; blocks or within a screen.
```

Comment:
To keep the comments inside the fill-column, could you please refill this to:
```
;; Define and default.  Both allow indentation for use in explicit
;; init blocks or within a screen.
```
(Just pressing M-q should do it if you are using the default key-bindings.)

Reply:
Sure, np.

(we really should setup something like a standard set of linters for these things)